zest/releaser/git.py

```diff
@@ -40,7 +40,7 @@ def name(self):
     def available_tags(self):
         tag_info = execute_command(['git', 'tag'])
         tags = [line for line in tag_info.split('\n') if line]
-        logger.debug("Available tags: %r", tags)
+        logger.debug("Available tags: '%s'", ' '.join(tags))
```

Comment:
Join with a comma+space?
OTOH, git prints them without...

Reply:
Ah, true, a comma would be clearer.
I'll change and locally and merge manually.